wherehows-data-model/DDL/ETL_DDL/etl_configure_tables.sql

```diff
@@ -13,82 +13,42 @@
 --
 
 -- configuration tables
-CREATE TABLE `wh_etl_job` (
-  `wh_etl_job_id`   INT(10) UNSIGNED  NOT NULL AUTO_INCREMENT
-  COMMENT 'id of the etl job',
-  `wh_etl_job_name` VARCHAR(127)      NOT NULL
+CREATE TABLE `wh_etl_job_schedule` (
+  `wh_etl_job_name` VARCHAR(127)  NOT NULL
   COMMENT 'etl job name',
-  `wh_etl_type`     VARCHAR(127)               DEFAULT NULL
-  COMMENT 'type of the etl service',
-  `cron_expr`       VARCHAR(127)               DEFAULT NULL
-  COMMENT 'frequency in crob expression',
-  `ref_id`          SMALLINT(6)       NOT NULL
-  COMMENT 'id of application/database where etl job runs',
-  `ref_id_type`     ENUM('APP', 'DB') NOT NULL
-  COMMENT 'flag the ref_id is an application or a database',
-  `timeout`         INT(11)                    DEFAULT NULL
-  COMMENT 'timeout in seconds for this etl job',
-  `next_run`        INT(10) UNSIGNED           DEFAULT NULL
+  `next_run` INT(10) UNSIGNED     DEFAULT NULL
   COMMENT 'next run time',
-  `comments`        VARCHAR(200)               DEFAULT NULL,
-  `cmd_param`       VARCHAR(500)               DEFAULT ''
-  COMMENT 'command line parameters for launch the job',
-  `is_active`       CHAR(1)                    DEFAULT 'Y'
-  COMMENT 'determine if this job is active or not',
-  PRIMARY KEY (`wh_etl_job_id`),
-  UNIQUE KEY `etl_unique` (`wh_etl_job_name`, `ref_id`)
+  PRIMARY KEY (`wh_etl_job_name`),
+  UNIQUE KEY `etl_unique` (`wh_etl_job_name`)
 )
   ENGINE = InnoDB
-  AUTO_INCREMENT = 1
-  DEFAULT CHARSET = utf8
-  COMMENT = 'WhereHows ETL jobs table';
+  DEFAULT CHARSET=utf8
+  COMMENT='WhereHows ETL job scheduling table';
 
-CREATE TABLE `wh_etl_job_execution` (
-  `wh_etl_exec_id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT
+CREATE TABLE `wh_etl_job_history` (
```

Comment:
It's probably better to use a different name here instead of doing schema evolution. Also if we screwed up, we can also rollback easily